app/uk/gov/hmrc/economiccrimelevyregistration/views/CheckYourAnswersView.scala.html

```diff
@@ -26,6 +26,8 @@
 @(eclDetails: SummaryList,
         organisationDetails: SummaryList,
         contactDetails: SummaryList,
+        otherEntityDetails: SummaryList,
+        hasOtherEntityData: Boolean,
```

Comment:
We do not need this one, you can use otherEntityDetails summaryList and instead of boolean check do something like this
!otherEntityDetails.rows.isEmpty

Reply:
Is there a `.isDefined` or `.nonEmpty` instead so we don't negate it?

Reply:
Oh yeah, for sure, it's a `Seq`,` .nonEmpty `would be perfect.